perfkitbenchmarker/vm_util.py

```diff
@@ -56,6 +57,8 @@
 flags.DEFINE_integer('burn_cpu_seconds', 0,
                      'Amount of time in seconds to burn cpu on vm.')
 flags.DEFINE_integer('burn_cpu_threads', 1, 'Number of threads to burn cpu.')
+flags.DEFINE_boolean('dstat', False, 'Run dstat on the VM during the benchmark '
```

Comment:
Can we give more details in the help message? For example, a short description of what dstat actually does. Honestly, I've never heard of dstat before until this project. :)

Maybe something like:

'Collects system resource metrics, for each VM during the execution of each benchmark'